python/lsst/meas/algorithms/ingestIndexManager.py

```diff
@@ -36,9 +36,9 @@
 
 # global shared counter to keep track of source ids
 # (multiprocess sharing is most easily done with a global)
-COUNTER = 0
+COUNTER = multiprocessing.Value('i', 0)
 # global shared counter to keep track of number of files processed.
-FILE_PROGRESS = 0
+FILE_PROGRESS = multiprocessing.Value('i', 0)
```

Comment:
Could you use the ctypes type (I think `c_int`, or maybe just `int`) instead of `'i'`` here? The `'i'` made me think that was the name of the counter or something.